app/views/projects/show.html.erb

```diff
@@ -166,9 +166,7 @@
           <div class="projects-collaborators-container">
             <div class="row center-row">
               <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 projects-collaborators-heading">
-                <div class="">
-               <%= t("projects.show.collaborators") %>
-                </div>
+                Collaborators
```

Comment:
Can you keep i18n version so that we can keep this translatable

Reply:
Sure!